setup.cfg

```diff
@@ -15,11 +15,13 @@ exclude =
 	./scripts,
 	__pycache__,
 	devenv.py
+        test_gke.py
```

Comment:
This will ignore `test_gke.py` only for `flake8` - coding style checks. Not needed imho.